### PR TITLE
SUL23-601: changed when focused to not be red with red underline

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- changed the focus to not have red text and underline. 

# Review By (Date)
- 8/29

# Criticality
- normal

# Urgency
- normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to https://su-library-git-sul23-601-focus-state-stanford-libraries.vercel.app/test-list-page
4. Verify the SUL people toggles no longer have red text/underline when it is focused.  It is supposed to have a red text hover and the red border.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SUL23-601 ](https://stanfordits.atlassian.net/browse/SUL23-601)
- 
# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/views/sul-people/sul-people-table-view.tsx [hunk 1]
```diff
@@ -244,7 +244,7 @@ const ToggleOption = ({
 
       <span
         className={twMerge(
-          "flex items-center whitespace-nowrap border border-black-80 p-4 pl-16 pr-32 text-14 leading-[30px] no-underline hover:border-cardinal-red-dark hover:bg-[#979694] hover:bg-opacity-10 hover:text-cardinal-red-dark hover:underline peer-checked:bg-[#979694] peer-checked:bg-opacity-20 peer-focus:border-2 peer-focus:border-black-80 peer-focus:bg-[#979694] peer-focus:bg-opacity-10 peer-focus:text-cardinal-red-dark peer-focus:underline md:text-16 peer-checked:[&_svg]:text-black",
+          "peer-focus:nounderline flex items-center whitespace-nowrap border border-black-80 p-4 pl-16 pr-32 text-14 leading-[30px] no-underline hover:border-cardinal-red-dark hover:bg-[#979694] hover:bg-opacity-10 hover:text-cardinal-red-dark hover:underline peer-checked:bg-[#979694] peer-checked:bg-opacity-20 peer-focus:border-2 peer-focus:border-black-80 peer-focus:bg-[#979694] peer-focus:bg-opacity-10 peer-focus:text-black-80 md:text-16 peer-checked:[&_svg]:text-black",
```
Comment (pookmish) [inline src/components/views/sul-people/sul-people-table-view.tsx:247]: The underline has to stay for focus. There is no indication that focus is on the element other than the border getting bigger.

**Review by pookmish — CHANGES_REQUESTED** [review]
This won't pass accessibility.

File: src/components/views/sul-people/sul-people-table-view.tsx (hunk 1 above)
Comment (jenbreese) [reply src/components/views/sul-people/sul-people-table-view.tsx:247]: Doh! I'm sorry; I'll fix. I really am not trying to be obtuse.

**Comment by jenbreese** [comment]
@pookmish I fixed the All to All specialists as requested in our Slack channel. I also fix the underline issue.

**Review by pookmish — APPROVED** [review]

**Pull request merged**